Make setting pages less dense without comments for each setting

hernanmd opened the pull request:
This PR includes two main changes to have better setting documentation in the New Settings Browser:

- Remove explicit documentation presenter below each setting. It makes loading settings a lot faster.
- Better class names for setting presenters.

<img width="1199" alt="Screenshot 2024-06-06 at 16 23 39" src="https://github.com/pharo-spec/NewTools/assets/4825959/fad2fa2f-2905-4310-b70e-55871671828f">

Commit:
Remove explicit documentation presenter below each setting. It makes loading settings a lot faster.
Better class names for setting presenters.

demarey commented:
I like it. It is better than the previous layout but now, it misses the documentation.
As discussed, you could define the right panel with `newPanedLayoutTopToBottom:` and add the documentation at the bottom side + set the position of slider to 80 percent by example.

hernanmd commented:
> I like it. It is better than the previous layout but now, it misses the documentation. As discussed, you could define the right panel with `newPanedLayoutTopToBottom:` and add the documentation at the bottom side + set the position of slider to 80 percent by example.

Yes, step by step :)
I will add it in another issue/PR so I don't make this one too big.

Commit:
Add a new presenter within the setting category page. 
When a setting input presenter is focused, the new presenter displays the currently selected setting label and its help text (if present).
Clean leftovers.

Ducasse commented:
```
StSettingsBrowserTest
 ✗ #testSimulateCategorySelection (42ms)
```

Commit:
Fix failing test
